packages/trip-form/i18n/fr.yml

```
@@ -39,6 +39,9 @@ otpUi:
     use: Utiliser
     useCompanies: Prestataires
     walkOnly: À pied uniquement
+  TripOptions:
+    transitOnly: Aller en transport en commun
```

"Transport" is used in plural form in French, and to save space, you may strip the verb "go" which is implied.
```suggestion
  transitOnly: En transports en commun
```

---

Thanks Binh, just updated to what you supplied.  Does the Spanish look ok to you?

---

It doesn't look obviously wrong, but you should confirm with a competent person if that is going on the TriMet site.

---

we don't actually use the non-english files from this project in our i18n implementation, but I ran it by a Spanish speaker from our team and they said it looks ok